flink-connector-examples/doc/flink-wordcount/README.md

```diff
@@ -58,7 +58,7 @@ In the windows where `WordCountReader` is running, it should show output similar
 
 ### Start Flink
 
-Follow the instruction [here](https://ci.apache.org/projects/flink/flink-docs-release-1.4/quickstart/setup_quickstart.html) to download and start Flink. 
+Follow the instruction [here](https://ci.apache.org/projects/flink/flink-docs-release-1.12/try-flink/local_installation.html) to download and start Flink.
```

Comment:
The link can use the latest stable version link, so it's not bind to a Flink version that require updates from time to time. Like https://ci.apache.org/projects/flink/flink-docs-stable/quickstart/setup_quickstart.html

Reply:
A latest version might also make the document unusable if Flink changes something in 1.1x. We'd better update only if a specific version works.

Reply:
Yes, we may update this link when the link changes.